INSTALL.md

```diff
@@ -66,11 +66,18 @@ mmdetection
 │   │   ├── train2017
 │   │   ├── val2017
 │   │   ├── test2017
+│   ├── cityscapes
+│   │   ├── annotations
+│   │   ├── train
+│   │   ├── val
 │   ├── VOCdevkit
 │   │   ├── VOC2007
 │   │   ├── VOC2012
 
 ```
+The cityscapes annotations have to be converted into the coco format using the [cityscapesScripts](https://github.com/mcordts/cityscapesScripts) toolbox.
+We plan to provide an easy to use conversion script. For the moment we recommend following the instructions provided in the 
+[maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark/tree/master/maskrcnn_benchmark/data) toolbox.
```

Comment:
@yhcao6 I think this is where the description about the file handling would have to go.

Reply:
I just move all images into one folder. You can add it into readme.

Reply:
Done!